targets/kodi

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
+    if release -le wheezy; then
+        error 99 "Kodi not supported on Debian Wheezy or older."
+    elif release -eq precise; then
+        error 99 "Kodi not supported on Precise Ubuntu."
+    fi
+fi
+REQUIRES='x11'
+PROVIDES='xbmc'
+DESCRIPTION='Installs the KODI media player. (Approx. 140MB)'
+HOSTBIN='startkodi'
+CHROOTETC='kodi-keyboard.xml kodi-cycle.py'
+. "${TARGETSDIR:="$PWD"}/common"
+
+### Append to prepare.sh:
+
+## Debian Overrides
+# Kodi wiki recommends using the Jessie Backports repository
+if release -eq jessie; then
+    cat > '/etc/apt/sources.list.d/kodi.list' <<EOF
+# Bring in Jessie Backports for newer Kodi
+deb http://http.debian.net/debian jessie-backports main
+EOF
+    # Update database
+    apt-get update || true
+fi
+
+## Ubuntu Overrides
+# Kodi has an official ppa for the latest stable version
+if [ "$DISTRO" -eq 'ubuntu' ]; then    
```

Comment:
ah, sorry, but string comparisons should stick to ='s.

`release` is a special command that converts the release name into a number and does numerical comparisons.